sys/psa_crypto/doc.txt

```diff
@@ -324,6 +324,12 @@
  * - psa_hash_sha_512_custom_backend
  * - psa_hash_sha_512_backend_riot
  *
+ * #### SHA 3
```

Comment:
This is probably the point where you could explicitly mention that this includes SHA3-256, SHA3-384 and SHA3-512.

Also, please move SHA 3 below the truncated SHA512 variants.